src/aws/resourcemanager/resourcemanager.py

```diff
@@ -1,16 +1,104 @@
 """
 Module for the Resource Manager.
 """
+import sys
+sys.path.append('./src')
```

Comment:
We do not need this on the master. This could be removed.

Reply:
Yeah forgot about that, will do.